util/src/main/java/com/farao_community/farao/util/SensitivitySecurityAnalysisService.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.util;
+
+import com.farao_community.farao.commons.FaraoException;
+import com.farao_community.farao.data.crac_api.*;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.contingency.BranchContingency;
+import com.powsybl.iidm.network.Branch;
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.TwoWindingsTransformer;
+import com.powsybl.sensitivity.*;
+import com.powsybl.sensitivity.factors.BranchFlowPerPSTAngle;
+import com.powsybl.sensitivity.factors.functions.BranchFlow;
+import com.powsybl.sensitivity.factors.variables.PhaseTapChangerAngle;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * @author Pengbo Wang {@literal <pengbo.wang at rte-international.com>}
+ */
+public final class SensitivitySecurityAnalysisService {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SensitivitySecurityAnalysisService.class);
+
+    private SensitivitySecurityAnalysisService() {
+    }
+
+    public static SensitivitySecurityAnalysisResult runSensitivity(Network network,
+                                                                   Crac crac,
+                                                                   ComputationManager computationManager) {
+        //prepare range actions: pst ( and hvdc in the future )
+        List<TwoWindingsTransformer> twoWindingsTransformers = getPstInRangeActions(network, crac.getRangeActions());
```

Comment:
Same for this you will be able to get rangeActions available on a specific state.

Reply:
OK. No action for now.